rules/DeadCode/Rector/ClassMethod/RemoveUnusedPrivateMethodRector.php

```diff
@@ -228,4 +234,34 @@ private function shouldSkipArrayCallable(Class_ $class, ?ArrayCallable $arrayCal
         // is current class method?
         return ! $this->isName($class, $arrayCallable->getClass());
     }
+
+    private function isPrivateAbstractMethodInTrait(Class_ $class, ?string $classMethodName): bool
+    {
+        if (null === $classMethodName) {
+            return false;
+        }
+
+        /** @var TraitUse[] $traitUsages */
+        $traitUsages = array_filter(
```

Comment:
please use AstResolver

https://github.com/rectorphp/rector-src/blob/e740a9388692804ea204b694459bdf1e1885165e/src/PhpParser/AstResolver.php#L286

Reply:
Good call, is the implementation at https://github.com/rectorphp/rector-src/pull/604/commits/ff805c85b13aaa3050136ea67cf3e8796c65255d how you imagined it? Or is there a "more generic" way to do it?

Reply:
Yes, it is good.